packages/toolbar/src/browser/style/toolbar.css

```diff
@@ -0,0 +1,253 @@
+/********************************************************************************
+ * Copyright (C) 2022 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+#main-toolbar {
+    --theia-toolbar-height: calc(var(--theia-private-menubar-height) - 2px);
+    --theia-toolbar-item-padding: 5px;
+    --theia-toolbar-icon-size: 20px;
+
+    min-height: var(--theia-toolbar-height);
+    color: var(--theia-mainToolbar-foreground);
+    background: var(--theia-mainToolbar-background);
+    padding: 2px 4px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    position: relative;
+}
+
+#main-toolbar .theia-progress-bar-container {
+    position: absolute;
+    bottom: 0;
+}
+
+.toolbar-wrapper {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    overflow: hidden;
+}
+
+.toolbar-wrapper .toolbar-column {
+    display: flex;
+    flex: 1;
+}
+
+.toolbar-wrapper .left {
+    justify-content: flex-start;
+}
+
+.toolbar-wrapper .center {
+    justify-content: center;
+}
+
+.toolbar-wrapper .right {
+    justify-content: flex-end;
+}
+
+.toolbar-wrapper:focus {
+    outline: none;
+}
+
+#main-toolbar .toolbar-item {
+    margin: 0;
+    padding: 0 var(--theia-toolbar-item-padding);
```

Comment:
Minor: This padding stretches the hover effect quite a bit in a horizontal direction and leaves no space above and below the buttons. Having `padding: 2px` and `margin: 0 2px` looks way better.